tools/pack_model_artifacts.py

```diff
@@ -0,0 +1,83 @@
+"""
+    Copyright 2025 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utils to package trained model artifacts and upload to S3 bucket
+"""
+
+import os
+import argparse
+from graphstorm.config import BUILTIN_TASK_NODE_CLASSIFICATION
+from graphstorm.config import BUILTIN_TASK_NODE_REGRESSION
+from graphstorm.config import BUILTIN_TASK_EDGE_CLASSIFICATION
+from graphstorm.config import BUILTIN_TASK_EDGE_REGRESSION
+from graphstorm.inference import wrap_model_artifacts
+
+SUPPORTED_TASKS = [BUILTIN_TASK_NODE_CLASSIFICATION,
+                   BUILTIN_TASK_NODE_REGRESSION,
+                   BUILTIN_TASK_EDGE_CLASSIFICATION,
+                   BUILTIN_TASK_EDGE_REGRESSION]
+
+def main(args):
+    """ Wrap model artifacts and entry point file into a compressed tar file.
+    """
+    # chose the target entry point file according to task
+    entry_base_path = 'python/graphstorm/sagemaker/deploy/'
+    if args.gml_task == BUILTIN_TASK_NODE_CLASSIFICATION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'node_classification_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_NODE_REGRESSION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'node_regression_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_EDGE_CLASSIFICATION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'edge_classification_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_EDGE_REGRESSION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'edge_regression_entry_point.py')
+    else:
+        raise NotImplementedError(f'Not supported GML task {args.gml_task}.')
```

Comment:
Usually better if we don't have strong assumptions about file structure, e.g. `entry_base_path = 'python/graphstorm/sagemaker/deploy/'`.

We can make `--entry-point` a required argument and just ask for the full path to it, as we do with the other files. This also allows users to define their own entry points.

Reply:
just remove this script as we don't it as a tool.